src/ert/gui/simulation/simulation_panel.py

```diff
@@ -109,7 +112,7 @@ def addSimulationConfigPanel(self, panel, mode_enabled: bool):
             item_count = self._simulation_mode_combo.count() - 1
             sim_item = self._simulation_mode_combo.model().item(item_count)
             sim_item.setEnabled(False)
-            sim_item.setToolTip("Disabled due to no observations")
+            sim_item.setToolTip("Disabled due to missing prerequisites")
```

Comment:
Perhaps be more specific, for example, "Disabled due to missing observations or parameters"?

Reply:
No problem!